# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1436836549 14400
#      Mon Jul 13 21:15:49 2015 -0400
# Node ID 22a2f6a7f6dcb309e8a2cb48787a390cf230f3a4
# Parent  3904d83e47d1c7cbe5fec1d19a9244111dc9ed3e
* scripts/module.mk (scripts-dist-hook): Always define.

diff --git a/scripts/module.mk b/scripts/module.mk
--- a/scripts/module.mk
+++ b/scripts/module.mk
@@ -186,16 +186,17 @@ install-pkg-add:
 
 uninstall-pkg-add:
 	for f in $(PKG_ADD_FILES); do \
 	  rm -f $(DESTDIR)$(fcnfiledir)/$$f; \
 	done
 .PHONY: uninstall-pkg-add
 
 if AMCOND_HAVE_JAVA
+scripts-dist-hook:
 else
 scripts-dist-hook:
 	@echo "Packaging distribution requires Java." ; exit 1;
 endif
 
 EXTRA_DIST += \
   $(SCRIPTS_IMAGES) \
   $(FCN_FILES) \
